packages/components/nodes/vectorstores/Milvus/Milvus.ts

```diff
@@ -1,45 +1,45 @@
-import { flatten } from 'lodash'
-import { DataType, ErrorCode, MetricType, IndexType } from '@zilliz/milvus2-sdk-node'
-import { Document } from '@langchain/core/documents'
-import { MilvusLibArgs, Milvus } from '@langchain/community/vectorstores/milvus'
-import { Embeddings } from '@langchain/core/embeddings'
-import { ICommonObject, INode, INodeData, INodeOutputsValue, INodeParams, IndexingResult } from '../../../src/Interface'
-import { getBaseClasses, getCredentialData, getCredentialParam } from '../../../src/utils'
+import { flatten } from 'lodash';
+import { DataType, ErrorCode, MetricType, IndexType } from '@zilliz/milvus2-sdk-node';
+import { Document } from '@langchain/core/documents';
+import { MilvusLibArgs, Milvus } from '@langchain/community/vectorstores/milvus';
+import { Embeddings } from '@langchain/core/embeddings';
+import { ICommonObject, INode, INodeData, INodeOutputsValue, INodeParams, IndexingResult } from '../../../src/Interface';
+import { getBaseClasses, getCredentialData, getCredentialParam } from '../../../src/utils';
 
 interface InsertRow {
-    [x: string]: string | number[]
+    [x: string]: string | number[];
 }
 
-class Milvus_VectorStores implements INode {
-    label: string
-    name: string
-    version: number
-    description: string
-    type: string
-    icon: string
-    category: string
-    badge: string
-    baseClasses: string[]
-    inputs: INodeParams[]
-    credential: INodeParams
-    outputs: INodeOutputsValue[]
+export class Milvus_VectorStores implements INode {
+    label: string;
+    name: string;
+    version: number;
+    description: string;
+    type: string;
+    icon: string;
+    category: string;
+    badge: string;
+    baseClasses: string[];
+    inputs: INodeParams[];
+    credential: INodeParams;
+    outputs: INodeOutputsValue[];
 
     constructor() {
-        this.label = 'Milvus'
-        this.name = 'milvus'
-        this.version = 1.0
-        this.type = 'Milvus'
-        this.icon = 'milvus.svg'
-        this.category = 'Vector Stores'
-        this.description = `Upsert embedded data and perform similarity search upon query using Milvus, world's most advanced open-source vector database`
-        this.baseClasses = [this.type, 'VectorStoreRetriever', 'BaseRetriever']
+        this.label = 'Milvus';
+        this.name = 'milvus';
+        this.version = 1.0;
```

Comment:
can we increment the version to `2.0`?

Reply:
You are using a newer version of Milvus SDK behind the scenes I don't see why not

Reply:
This is the version that is used to indicate the component version on Flowise, usually we update the version whenever there is new changes made to the component